circle.yml

```diff
@@ -225,6 +225,41 @@ jobs:
            name: Test
            command: |
              test -f out/bin/ctags && ( file out/bin/ctags | grep -q 'ARM aarch64' )
+   centos6:
+     working_directory: ~/universal-ctags
+     docker:
+       - image: centos:6
+     steps:
+       - run:
+           name: Rearrange CentOS-Base.repo file
+           command: |
+             sed -i "s|#baseurl=|baseurl=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|http://mirror\.centos\.org/centos/\$releasever|https://vault\.centos\.org/6.10|g" /etc/yum.repos.d/CentOS-Base.repo
+       - run:
+           name: Install tools for buildig
+           command: |
+             yum install -y git gcc make autconf automake pkgconfig
```

Comment:
`autconf ` spelling mistake. Actually, `autoconf` will installed along with `automake`

Reply:
Thank you!